src/items/LinkItem.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { EmbeddedLinkItemExtra, ImmutableItemClass } from '../types';
+import { TABLE_CATEGORIES } from '../utils/storybook';
+import LinkItem from './LinkItem';
+import { Item, ItemType } from '@graasp/sdk';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { RecordOf } from 'immutable';
+import React from 'react';
```

Comment:
This repo is not using the sorted imports plugin ? We could add it (maybe in a next PR)

Reply:
I've added it in the mui5 branch 👍